examples/tts/conf/fastpitch/fastpitch.yaml

```diff
@@ -193,6 +193,8 @@ model:
   alignment_module:
     _target_: nemo.collections.tts.modules.aligner.AlignmentEncoder
     n_text_channels: ${model.symbols_embedding_dim}
+    dist_type: cosine
+    temperature: 15.0
```

Comment:
any guidance on the empirical number?

Reply:
I just found ~15 to work well by testing multiples of 5 (I assume the L2 distance temperature of 0.0005 was similarly found by trial and error). Its been about a year since I experimented with it, but I vaguely recall 30 not working and 10 converging too slow.